srtcore/api.cpp

```diff
@@ -1356,16 +1499,20 @@ int CUDTUnited::epoll_release(const int eid)
    return m_EPoll.release(eid);
 }
 
-CUDTSocket* CUDTUnited::locate(const SRTSOCKET u)
+CUDTSocket* CUDTUnited::locateSocket(const SRTSOCKET u, ErrorHandling erh)
```

Comment:
The reason why `locate` was renamed to `locateSocket` was first of all because there will be another function added, `locateGroup`.

Also the `lookup` function has been removed because it duplicates the functionality of `locate` and the only difference was the way of informing about the failure: `locate` returned error, `lookup` threw exception. These two things have been now made common by `locateSocket` which's error handling method is selected through the additional parameter.